### PR TITLE
Feature/kx 97 dynamically customize dashboard

### PR DESCRIPTION
**Pull request by GiuseppeArbore**
Please suggest a color if we want to use a color for the column with the arrow

### CODE REVIEW

File: client/src/components/Console.tsx [hunk 1]
```diff
@@ -97,6 +97,14 @@ export default function Console() {
     }
   }, [selectedView, refreshNeeded]);
 
+  const [showSideBar, setShowSideBar] = useState(true);
```
Comment (golnazsaraji) [inline client/src/components/Console.tsx:100]: Perhaps a little margin on the left side can be better

File: client/src/components/Console.tsx [hunk 2]
```diff
@@ -126,44 +134,73 @@ export default function Console() {
...
 
+        </Col>
         <Col numColSpanLg={1}>
-          <div className="space-y-6">
-            <FormDialog
-              documents={documents}
-              refresh={() => setRefreshNeeded(true)}
-            />
+          <div className="flex flex-row ">
+            {showSideBar &&
```
Comment (golnazsaraji) [inline client/src/components/Console.tsx:156]: Perhaps #3A7CA5 can be more relevant!

**Review by golnazsaraji — APPROVED** [review]
grazie!

**Review by AlbertoPedalino — DISMISSED** [review]
image
if the column is closed this happen

**Comment by AlbertoPedalino** [comment]
image
for concistency i think we need that bar also with this layout

**Comment by AlbertoPedalino** [comment]
i suggest this color for the bar color [#e7ebf3](https://www.color-hex.com/color/e7ebf3)
image

File: client/src/components/Console.tsx (hunk 2 above)
Comment (GiuseppeArbore) [reply client/src/components/Console.tsx:156]: thanks for your suggestion!

**Comment by GiuseppeArbore** [comment]
> image if the column is closed this happen

Trying it on the PC it does not happen while on the external monitor it does. I think it is to improve the behavior of the list so that it fits better on the screen (kx-78 if I'm not mistaken).

File: client/src/components/Console.tsx (hunk 1 above)
Comment (GiuseppeArbore) [reply client/src/components/Console.tsx:108]: added a bit when closed

File: client/src/components/Console.tsx [hunk 3]
```diff
@@ -132,44 +158,73 @@ export default function Console() {
...
+              <Col className="hider mr-1 hide-on-small" role="Button">
```
[the remark this answers is not in the document]
Comment (GiuseppeArbore) [reply client/src/components/Console.tsx:181]: great suggestion!

File: client/src/components/Console.tsx [hunk 4]
```diff
@@ -132,44 +158,73 @@ export default function Console() {
...
+                <i className="h-full" onClick={() => setShowSideBar(false)}><RiArrowRightSLine className="h-full" ></RiArrowRightSLine></i>
```
[the remark this answers is not in the document]
Comment (GiuseppeArbore) [reply client/src/components/Console.tsx]: done

**Pull request merged**